util/db/repository_test.go

```diff
@@ -24,9 +24,7 @@ func TestRepoURLToSecretName(t *testing.T) {
 }
 
 func Test_getRepositoryCredentialIndex(t *testing.T) {
-	argoCDSettings := settings.ArgoCDSettings{
-		RepositoryCredentials: []settings.RepoCredentials{{URL: "http://known"}},
-	}
+	repositoryCredentials := []settings.RepoCredentials{{URL: "http://known"}}
```

Comment:
does this test belong in this package anymore?

Reply:
I think it does. It tests function defined in repository.go .